unit_test/main_tests/conftest.py

```diff
@@ -70,5 +70,14 @@ def platform(request, monkeypatch):
 @pytest.fixture
 def intercepted_build_args(platform, monkeypatch):
     intercepted = ArgsInterceptor()
-    monkeypatch.setattr(globals()[platform], 'build', intercepted)
+
+    if platform == 'linux':
```

Comment:
What happened here? Teah, it's probably cleaner this way, but just curious if the old approach broke something?

Reply:
I just found the old approach a bit surprising, when I was using it! So I made it more explicit.